inference-engine/tests/functional/plugin/shared/include/single_layer_tests/mvn_6.hpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
IMO, MVN6 and MVN should be together. I mean, mv6 parameters, tests classes, builders and so o should be in `mvn` source file

Reply:
FIxed